### PR TITLE
build: put version stamp to build artifacts

### PR DESCRIPTION
**Pull request by thesayyn**

### CODE REVIEW

File: scripts/current_version.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# Used with Bazel's stamping feature
+echo BUILD_SCM_VERSION ${BUILD_SCM_VERSION_OVERRIDE:-$(git describe --abbrev=7 --tags HEAD)}
```
Comment (emre2345) [inline scripts/current_version.sh]: @thesayyn Based on https://git-scm.com/docs/git-describe default value of --abbrev is 7 and it's used to change the default value. This means we might remove it if there's not any special reason for you to put it there.

**Comment by thesayyn** [comment]
I couldn't get it what you mean exactly by changing the reason?

**Comment by emre2345** [comment]
> I couldn't get it what you mean exactly by changing the reason?

Sorry. I wanted to ask the reason of the change :)

**Comment by thesayyn** [comment]
I did those changes because instead forcing the users directly to https protocol, we should let the browser decide whether the connection should be established through http or https.

**Comment by emre2345** [comment]
> It's did those changes because instead forcing the users directly to https protocol, we should let the browser decide whether the connection should be established through http or https.

I see, thanks.

File: scripts/current_version.sh (hunk 1 above)
Comment (thesayyn) [reply scripts/current_version.sh]: It might be different in older versions of git. This is the reason why I have put it there.

Comment (emre2345) [reply scripts/current_version.sh]: Makes sense. Thanks.

**Pull request merged**